### ISSUE TITLE
os.sched_setaffinity(0, [2*(cpu_offset+rank)]) OSError: [Errno 22] Invalid argument

### ISSUE THREAD

**Issue by LukeLIN-web** [quiver-team/torch-quiver#162]
When I run torch-quiver/srcs/python/quiver/serving.py
It shows 
```
Process Process-29:
Traceback (most recent call last):
  File "/opt/conda/lib/python3.10/multiprocessing/process.py", line 314, in _bootstrap
    self.run()
  File "/opt/conda/lib/python3.10/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/root/share/torch-quiver/srcs/python/quiver/serving.py", line 120, in cpu_sampler_worker_loop
    os.sched_setaffinity(0, [2*(cpu_offset+rank)])
OSError: [Errno 22] Invalid argument
```

The source code is 

```
    def cpu_sampler_worker_loop(self, rank, sample_task_queue_list, result_queue_list, device_num, sizes, csr_topo, cpu_offset):
        os.sched_setaffinity(0, [2*(cpu_offset+rank)])
```

I have only 16 CPU
```
lscpu
Architecture:        x86_64
CPU op-mode(s):      32-bit, 64-bit
Byte Order:          Little Endian
CPU(s):              16
On-line CPU(s) list: 0-15
```

**Comment by LukeLIN-web** [comment]
When I mute all ` os.sched_setaffinity(0, [2*(cpu_offset+rank)])`, run torch-quiver/srcs/python/quiver/serving.py, it will get stuck, hanging after printing many results
```
tensor([[-3.7092, -4.0264, -3.7144,  ..., -3.7409, -3.5885, -3.7897],
        [-3.7121, -4.0138, -3.8532,  ..., -3.8745, -3.6802, -3.5403],
        [-3.4066, -3.9946, -3.7638,  ..., -3.7612, -3.7018, -3.4789],
        ...,
        [-3.7253, -4.1716, -4.0981,  ..., -3.7491, -3.5588, -3.5788],
        [-3.8461, -3.7991, -3.8211,  ..., -3.8962, -3.6386, -3.5820],
        [-3.7833, -4.1650, -3.5723,  ..., -4.2274, -3.4586, -3.4708]])
```

**Comment by CongjieHe** [comment]
This bug has been fixed in #164. We corrected the calculation method for the CPU range in reddit_serving.py and provided a cpu_range parameter for easy customization of the process-to-CPU binding mapping.

**Comment by LukeLIN-web** [comment]
> This bug has been fixed in #164. We corrected the calculation method for the CPU range in reddit_serving.py and provided a cpu_range parameter for easy customization of the process-to-CPU binding mapping.

It's great work! Thank you for your time!
